lld/ELF/ICF.cpp

```diff
@@ -561,11 +598,25 @@ template <class ELFT> void ICF<ELFT>::run() {
           d->folded = true;
         }
   };
-  for (Symbol *sym : ctx.symtab->getSymbols())
+  for (Symbol *sym : ctx.symtab->getSymbols()) {
     fold(sym);
+    auto it = symbolEquivalence.findLeader(sym);
+    if (it != symbolEquivalence.member_end() && *it != sym) {
+      print() << "Redirecting " << sym->getName() << " to " << (*it)->getName();
```

Comment:
Use lower-case messages (convention; we follow https://llvm.org/docs/CodingStandards.html#error-and-warning-messages)

In new messages, we quote symbol names like `'sym'`